.gitignore

```diff
@@ -8,3 +8,4 @@ papers
 tmp
 *.iml
 .coverage
+*.code-workspace
```

Comment:
Line feed at the end of file is a good habit

( https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline )